Makefile

```diff
@@ -0,0 +1,2 @@
+all:
```

Comment:
add a new Makefile. if we decide to use Makefile. I sugguest to remove build.sh. but we need implement all the things that build.sh already implemented.

the build.sh also build the cgitest.cgi, so makefile should do it.
another way is using uname to check the building enviroment in build.sh.

OS_TYPE=`uname`
OS_TYPE=${OS_TYPE,,}

if [[ $OS_TYPE == *linux* ]]; then
# todo...
fi

if [[ $OS_TYPE == *msys* ]]; then
# todo...
fi

Reply:
Ok I will update this PR with a more complete Makefile